Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/RepositoryItemUsagePage.xaml.cs

```diff
@@ -293,12 +294,16 @@ private async void UpdateAllButton_Click(object sender, RoutedEventArgs e)
                     {
                         if (usage.Status == RepositoryItemUsage.eStatus.Pending)
                         {
-
                             if (usage.HostActivity != null)
-                                mRepoItem.UpdateInstance(usage.UsageItem, usage.SelectedItemPart, usage.HostActivity);
+                            {
+                                this.Dispatcher.Invoke(() => mRepoItem.UpdateInstance(usage.UsageItem, usage.SelectedItemPart, usage.HostActivity));
+                                //mRepoItem.UpdateInstance(usage.UsageItem, usage.SelectedItemPart, usage.HostActivity);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4582993)